hw/ip/entropy_src/doc/theory_of_operation.md

```diff
@@ -148,13 +148,13 @@ However, if the ENTROPY_SRC block experiences internal back pressure, health tes
 The reduce the probability of dropping post-health test entropy bits, the **Distribution FIFO** can be used.
 This FIFO has pass-through mode enabled meaning it doesn't add latency to hardware pipeline.
 It has a width of 32 bits.
-Its depth is configurable via compile-time Verilog parameter and should matched to the expected level of conditioner back pressure.
+Its depth is configurable via compile-time Verilog parameter and should be matched to the expected level of conditioner back pressure.
```

Comment:
We could possibly even write:
`Its depth is configurable via compile-time Verilog parameter and should match the expected level of conditioner back pressure.`